hw03_glek.py

```diff
@@ -0,0 +1,92 @@
+#################################################################################
+# Author: Kafui Gle
+# Username: glek
+#
+# Assignment: hw03
+# Purpose:
+# Google Doc Link: https://docs.google.com/document/d/1yGLLdFBhjPw96O7yJiaUsUuLIPZWXuOvJ6QDpzMWM_c/edit
```

Comment:
Use the header block formatting provided to you in the template file.

Reply:
changed the header block formatting style to the one in the template file as suggested.